web_drop_target/__manifest__.py

```diff
@@ -3,14 +3,15 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 {
     "name": "Drop target support",
-    "version": "10.0.1.0.0",
+    "version": "10.0.1.1.0",
     "author": "Therp BV,Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/web",
     "license": "AGPL-3",
     "category": "Usability",
     "summary": "Allows to drag files into Odoo",
     "depends": [
         'web',
+        'document'
```

Comment:
do we really need this?

Reply:
I think yes, documents module adds the attachment button on top of the forms.

Reply:
that's a functional thing. Is it *technically* necessary?

Reply:
not really AFAIS but it sucks when you add an attachment and you don't see it...
If you don't want the dependency,  maybe we can add a reminder in  the drop area saying that you won't see them unless...?

Reply:
that's worse in my opinion. I think technically unnecessary dependencies are a bad habit because you never know what people out there do with your code, and then pulling a dependency that might or might not cause trouble can be annoying. But if you both insist, I won't block on this

Reply:
Well, I don't think that this dependency is unnecesary, if you don't install this module you can't see the attachments. If you want I can delete it, no problem, but we must add in the readme a message warning of this.

Reply:
just leave it as it is, there's more interesting stuff to do and to discuss here